site/en/docs/devtools/css/reference/index.md

```diff
@@ -128,6 +128,34 @@ If your browser supports the `lab()` function, the element is green, otherwise i
 **Note**: At the time of writing, only Safari [supports the CIELAB color space](https://caniuse.com/?search=lab).
 {% endAside %}
 
+### View `@scope` at-rules {: #scope }
+
+The **Styles** pane shows you [CSS `@scope` at-rules](https://drafts.csswg.org/css-cascade-6/#scope-atrule) if they are applied to an element.
+
+The new `@scope` at-rules are a part of the [CSS Cascading and Inheritance Level 6 specification](https://drafts.csswg.org/css-cascade-6/). These rules allow you to scope CSS styles, in other words, explicitly apply styles to specific elements.
+
+{% Aside %}
+**Note**: The `@scope` at-rule feature is experimental. To test it, enable the **Experimental Web Platform features** flag in `chrome://flags/#enable-experimental-web-platform-features`. Otherwise, the preview below doesn't work.
+{% endAside %}
+
+View the `@scope` rule in the following preview:
+
+{% Codepen {
+ user: 'sofiayem',
+ id: 'YzaBqOo',
+ height: 270,
+ allow: ['geolocation']
```

Comment:
curious, why the demo need geolocation access?

Reply:
Actually, I don't know. I copied an example from the handbook.